### ISSUE TITLE
Model exported to keras gets very bad evalution score

### ISSUE THREAD

**Issue by kayfg** [jhfjhfj1/autokeras#308]
I already posted my issue in the gitter autokeras lobby and on stackoverflow but I try it here again. Link to detailed version : https://stackoverflow.com/questions/53229283/autokeras-exported-model-performs-not-as-expected

Keras version: 2.2.2
Tensorflow version: 1.12.0

So I train with a dataset of (n, 50, 50, 3) images and get models with high accuracy(0.99) and low loss(0.02). I followed the tutorial on https://autokeras.com/start/.
After final_fit I export my model with
clf.load_searcher().load_best_model().produce_keras_model().save('my_model.h5')
and load it in keras but I get very bad evaluation scores. Accuracy is at 0.12 and loss at 5.1.
Can anyone help me? I guess it is just some command I need to use before exporting or after loading to get my model working in keras.

**Comment by sethuiyer** [comment]
I believe this has to do with data transformation issues. Try using `export_autokeras_model` instead.

**Comment by kayfg** [comment]
You mean clf.export_autokeras_model?
How can I transform this to a keras model after the export?
I can't load the autokeras model.

Unfortunately there is no alternative to
clf.load_searcher().load_best_model().produce_keras_model().save('my_model.h5')
with produce autokeras model.

**Comment by jhfjhfj1** [comment]
It is not possible to export the entire data pipeline to a Keras model.
Only the neural network is exported.
Without the data preprocessing, the Keras model won't perform well.
